src/components/Tweetbox.tsx

```diff
@@ -1,21 +1,50 @@
-import React from "react";
+import React, { useState } from "react";
 import { Avatar, Button } from "@material-ui/core";
 import "./Tweetbox.css";
+import db from "../firebase";
 
-const Tweetbox = () => {
+type TweetboxProps = {};
+
+const Tweetbox: React.FC<TweetboxProps> = () => {
```

Comment:
If there are no props, use just `React.FC` , TweetboxProps is not needed here

Reply:
Resolved